package.json

```diff
@@ -4,7 +4,7 @@
 	"description": "GitHub integration for Atlassian Jira",
 	"repository": "https://github.com/atlassian/github-for-jira.git",
 	"engines": {
-		"node": ">= 14.21 <15",
+		"node": ">= 14.17 <15",
```

Comment:
Just curious why?
/nit

Reply:
because there was an issue last week with the snyk scanner which didn't have 14.21 available to do.  Not sure if it was temporary or not, but it completely blocked deployment, so this was the fix as it did have 14.17 available